code/modules/reagents/Chemistry-Holder.dm

```diff
@@ -901,26 +931,19 @@ trans_to_atmos(var/datum/gas_mixture/target, var/amount=1, var/multiplier=1, var
 /datum/reagents/proc/is_full()
 	return total_volume >= maximum_volume
 
-/datum/reagents/proc/get_heatcapacity()
-	var/heat_capacity = 0
-
-	if(reagent_list.len)
-		for(var/datum/reagent/R in reagent_list)
-			heat_capacity += R.volume*R.specheatcap
-
-	return heat_capacity
-
 /datum/reagents/proc/get_overall_mass()
 	//M = DV
-
 	var/overall_mass = 0
-
-	if(reagent_list.len)
-		for(var/datum/reagent/R in reagent_list)
-			overall_mass += R.density*R.volume
-
+	for(var/datum/reagent/R in reagent_list)
+		overall_mass += R.density*R.volume
 	return overall_mass
 
+/datum/reagents/proc/get_thermal_mass()
```

Comment:
I still don't get the point of changing this name

Reply:
might be less confusing(at least it is for me)
if you look down a bit  there used to be an instance of `mass * heat_capacity` which resulted in bogus units, which usually happens when people get confused with their units

heat capacity $[\frac{J}{K}]$ aka thermal mass applies to the whole object
specific heat capacity $[\frac{J}{Kg K}]$ is a material property
there's also about 5 more names for each of the 2 because why not LOL

Reply:
Yeah I think we often use "heat capacity" and "specific heat capacity" somewhat interchangably if they aren't, so "thermal mass" makes a bit less ambiguous that we're talking about the true meaning of "heat capacity" and not a shorthand for specheatcap.

Reply:
The normal shorthand for specific heat capacity is specific heat, but I guess I could see how the terms could be confusing to someone who hasn't been steeped in physics for a decade